src/components/RewardsCard/index.module.scss

```diff
@@ -0,0 +1,49 @@
+.cardBase {
+  width: 271px;
+  // should be 379px when we update it to use two stickers
+  height: 339px;
+  flex-shrink: 0;
+  justify-content: space-between;
+  background-color: var(--color-neutral-600);
+  border-radius: var(--space-xs);
+}
+
+.rewardsCard {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  width: 100%;
+  justify-content: flex-end;
+  max-height: 108px;
+}
+
+.content {
+  text-align: center;
+}
+
+.rewardImage {
+  position: relative;
+  width: 100%;
+  height: 271px;
```

Comment:
could we make this more dynamic than just hardcoding a height?

Reply:
Same issue as above:
![image](https://github.com/user-attachments/assets/6b21baca-f35b-4737-be3d-029e55cf9cb5)

Reply:
Another point actually is that if we make it dynamic we can have cards with different sizes depending on the content. Both on mobile and desktop they have the same size, unless something changed last time I checked there.